### ISSUE TITLE
axis argument torch.stack is undocumented

### ISSUE THREAD

**Issue by janden** [kymatio/kymatio#497]
We should use `dim` instead, per https://pytorch.org/docs/stable/torch.html#torch.stack.

**Comment by edouardoyallon** [comment]
I think it's removed everywhere now via the PR there

**Comment by edouardoyallon** [comment]
Now it's good thanks to @eugenium

**Comment by eugenium** [comment]
I only edited it in the sklearn pull shouldnt we wait till its in 0.2.x

**Comment by edouardoyallon** [comment]
can you do a separate PR?

**Comment by edouardoyallon** [comment]
I think this is down, I've done grep 'torch.stack' ./*.py and couldn't find anything..

**Comment by edouardoyallon** [comment]
@janden we close, right?

**Comment by edouardoyallon** [comment]
I confirm this is removed.
